generators/app/templates/package.json

```diff
@@ -24,6 +25,7 @@
     "i18next-xhr-backend": "^2.0.1",
     "identity-obj-proxy": "3.0.0",
     "moment": "^2.24.0",
+    "n3": "^1.3.5",
```

Comment:
Why is this library being pulled in (i.e. I don't see any need in the rest of this PR)...?

Reply:
Good catch! Was part of debugging, will remove